### PR TITLE
webapp: Appservice plan update supports Isolated SKU

### LINKED ISSUE #5760: webapp:Support App Service Environment Isolated Plan SKUs for scaling-up/down an App Service Plan
Support App Service Isolated Plan SKU, i.e. instance types (I1, I2, I3) for App Service Plan scaling, e.g. az appservice plan update --sku I2  ...

Today, it is not possible to use the Azure CLI to scale-up/down instances in App Service Environment.

### PR DESCRIPTION
**Pull request by panchagnula**
Fixes #5760
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by panchagnula** [comment]
@qwordy can i get a PR on this please? Thank you.

**Comment by panchagnula** [comment]
@qwordy can I get a PR please - this already missed the original release milestone because of the delay in PR. Thank you.

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -55,7 +61,6 @@ Release History
 
 **AppService**
 
-* Added "webapp config access-restriction show | set | add | remove"
```
Comment (qwordy) [inline src/azure-cli/HISTORY.rst]: I'm not sure whether we should modify history of old versions.

**Comment by panchagnula** [comment]
The reason i added no test cases is because this SKU supported on ASE resource types only & this is not something we currently support on CLI. We just added the option to scale an ASP using CLI for now.

**Pull request merged**